processing/src/main/java/io/druid/query/groupby/epinephelinae/GroupByQueryEngineV2.java

```diff
@@ -58,6 +64,19 @@
 
 public class GroupByQueryEngineV2
 {
+  private static final GroupByStrategyFactory STRATEGY_FACTORY = new GroupByStrategyFactory();
+
+  private static GroupByColumnSelectorPlus[] getGroupBySelectorPlus(ColumnSelectorPlus<GroupByColumnSelectorStrategy>[] baseSelectorPlus)
```

Comment:
`createGroupBySelectorPlus` is more appropriate, since we're creating not getting.